.github/workflows/run-tests.yml

```
@@ -67,6 +68,24 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+
+      - name: Install Anchor CLI (if cache miss)
+        if: steps.cache-anchor.outputs.cache-hit != 'true'
```

actionlint (1.7.8)

property "cache-anchor" is not defined in object type {}

(expression)